_pages/pt/developer-engagement/webcompat-sprint.md

```diff
@@ -6,69 +6,69 @@ lang: pt
 
-Web Compatibility, the ability for all Web content to run in all browsers, is important. There are multiple web content engines running on multiple devices, and sometimes with a little testing and some small tweaks to their site, developers can ensure that more users can have the full experience that is desired.
+Para mais informação e dados sobre Compatibilidade Web, leia o [recente blog post de Mozilla Hacks](https://hacks.mozilla.org/2016/07/make-the-web-work-for-everyone/)
 
-For more background and data on Web Compatibility, read the [recent blog post on Mozilla Hacks](https://hacks.mozilla.org/2016/07/make-the-web-work-for-everyone/)
+## Impacto e Métricas
 
-## Impact and Metrics
+Na Mozilla, queremos que a web funcione bem para todos, não apenas em um ou dois navegadores. Isto significa cross-browser e cross-device. Nossa comunidade global pode dar visibilidade a isso que nenhum outro time faria!
 
-At Mozilla, we want the web running well for everyone, not just in one or two browsers. This means cross-browser, and cross-device. Our global community can give visibility to this that no team could ever match!
-
-### Goals for this area
+### Objetivos para esta área
 
 __Impact:__
 
-* 300 Add-on installs
-* 200 compatibility issues filed
+* 300 Complementos instalados
+* 200 Problemas de compatibilidade arquivados
 
 __Strength:__
 
-* 50 Mozillians run events
-* 50 signups to the [compatibility list](https://www.mozilla.org/en-US/about/forums/#compatibility)
+* 50 Mozillians organizando eventos
+* 50 assinaturas na [lista de compatibilidade](https://www.mozilla.org/en-US/about/forums/#compatibility)
 
-## Audience for this activity
+## Audiência para esta atividade
 
-The audience this activity is anyone who is passionate on the topic, but being somewhat Web tech savvy is a help. No programming skills are necessary. It is for people who care about diversity on the Web.
+A audiência desta atividade é alguém que é apaixonado sobre o tema, mas sendo um "técnico" web é uma grande ajuda. Nenhuma habilidade de programação é necessária. É para pessoas que se preocupam com a diversidade na Web.
 
-## Activity Format
+## Formato de atividade
 
-While this activity can be run solo, we encourage you to run an event in your community to get together people who care about the Web. The goal is to test sites, diagnose problems, and discuss how to evangelise locally to developers about a Web that works for everyone.
+Essa atividade pode ser executada sozinho, mas nós encorajamos você a executar um evento em sua comunidade para reunir as pessoas que se preocupam com a Web. O objetivo é testar sites, diagnosticar problemas e discutir como evangelizar localmente para desenvolvedores sobre uma Web que funcione para todos.
```

Comment:
"Você pode executar essa atividade sozinho, [...] e discutir como evangelizar os desenvolvedores locais sobre uma Web que funcione para todos."
Ainda to em dúvida sobre `discutir como evangelizar os desenvolvedores locais`, mas acho que ficou melhor :)